addon/-private/data-view/radar/radar.js

```diff
@@ -207,24 +209,33 @@ export default class Radar {
     assert('itemContainer must be set on Radar before scheduling an update', itemContainer !== null);
     assert('scrollContainer must be set on Radar before scheduling an update', scrollContainer !== null);
 
-    const {
-      top: itemContainerTop
-    } = itemContainer.getBoundingClientRect();
-    const {
-      top: scrollContainerTop,
-      height: scrollContainerHeight
-    } = scrollContainer.getBoundingClientRect();
+    // The scroll container's offsetHeight will reflect the actual height of the element, while
+    // it's measured height via bounding client rect will reflect the height with any transformations
+    // applied. We use this to find out the scale of the items so we can store measurements at the
+    // correct heights.
+    const scrollContainerOffsetHeight = scrollContainer.offsetHeight;
+    const { height: scrollContainerRenderedHeight } = scrollContainer.getBoundingClientRect();
+
+    // Represents the opposite of the scale, if any, applied to the collection
+    const transformScale = scrollContainerOffsetHeight > 0 ? scrollContainerOffsetHeight / scrollContainerRenderedHeight : 1;
+
+    const { top: itemContainerTop } = getScaledClientRect(itemContainer, transformScale);
+    const { top: scrollContainerTop } = getScaledClientRect(scrollContainer, transformScale);
 
     let maxHeight = 0;
+
     if (scrollContainer instanceof Element) {
-      maxHeight = estimateElementMaxHeight(scrollContainer);
-    }
+      const elementMaxHeight = window.getComputedStyle(scrollContainer).maxHeight;
```

Comment:
will this break in fastboot?

Reply:
It should not, we should never get to this function in fastboot. We render solely based on `initialRenderCount` and never schedule anything. I think currently that's based on the fact that RAF doesn't work in fastboot, it should be more explicit. That could actually be a part of the other PR, since it's ultimately about normalizing "starting" the Radar.

Reply:
Ah, actually it is explicit because we only do that in `didInsertElement`, which does not run in Fastboot